templates/example_cell_clustering.ipynb

```diff
@@ -17,7 +17,9 @@
   {
```

Comment:
<p>Sorry Alex, I think I told you before to specify cell_cluster_col and calculate the weighted average for either cell_som_cluster or cell_meta_cluster based on the user-specified variable, but actually we need to find the weighted average for both. For Zak's visualization, we need both.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/460/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>Ok no worries, I'll change this up to support both visualizations.</p>